scu-es/src/fiskaltrust.Middleware.SCU.ES.VeriFactu/VeriFactuConfiguration.cs

```diff
@@ -11,8 +11,20 @@ public class VeriFactuSCUConfiguration
         public string QRCodeBaseUrl { get; set; } = "https://prewww2.aeat.es";
 
         public X509Certificate2 Certificate { get; set; } = null!;
+        public string Nif { get; set; } = null!;
+
+        public string NombreRazonEmisor { get; set; } = null!; 
 
         public static VeriFactuSCUConfiguration FromConfiguration(Dictionary<string, object> configuration)
-            => JsonSerializer.Deserialize<VeriFactuSCUConfiguration>(JsonSerializer.Serialize(configuration)) ?? new VeriFactuSCUConfiguration();
+        {
+            return new VeriFactuSCUConfiguration
+            {
+                BaseUrl = configuration["BaseUrl"].ToString()!,
+                QRCodeBaseUrl = configuration["QRCodeBaseUrl"].ToString()!,
+                Certificate = (X509Certificate2)configuration["Certificate"]!,
+                Nif = configuration["Nif"].ToString()!,
+                NombreRazonEmisor = configuration["NombreRazonEmisor"].ToString()!,
+            };
+        }
```

Comment:
We don't need the `VeriFactuSCUConfiguration FromConfiguration(Dictionary<string, object> configuration)` method anymore I think since we don't load the config from the cashboxconfiguration anymore